toolkit/tools/pkg/imagecustomizerlib/artifactsinputoutput.go

```diff
@@ -73,8 +104,18 @@ func outputArtifacts(items []imagecustomizerapi.OutputArtifactsItemType, outputD
 				destPath := filepath.Join(outputDir, entry.Name())
 				err := file.Copy(srcPath, destPath)
 				if err != nil {
-					return fmt.Errorf("failed to copy binary from (%s) to (%s):\n%w", srcPath, destPath, err)
+					return nil, fmt.Errorf("failed to copy binary from (%s) to (%s):\n%w", srcPath, destPath, err)
 				}
+
+				signedName := replaceSuffix(entry.Name(), ".unsigned.efi", ".signed.efi")
+				source := prefix + filepath.Join(configOutputArtifactsPath, signedName)
+				destinationName := replaceSuffix(entry.Name(), ".unsigned.efi", ".efi")
+
+				outputArtifactsMetadata = append(outputArtifactsMetadata, OutputArtifactMetadata{
+					Partition:   partition,
+					Source:      source,
+					Destination: filepath.Join("/EFI/Linux", destinationName),
```

Comment:
Use `UkiOutputDir` instead of `"/EFI/Linux"`.

You can write this as `filepath.Join("/", UkiOutputDir, destinationName)`